apps/outlook/src/pages/accounts/account-card.tsx

```diff
@@ -0,0 +1,242 @@
+import type { RouterOutputs } from "@agreeto/api";
+import { EventColorUserRadix } from "@agreeto/api/types";
+import {
+  Button,
+  Dialog,
+  DropdownMenu,
+  GoogleLogo,
+  MicrosoftLogo,
+} from "@agreeto/ui";
+import * as RadioGroup from "@radix-ui/react-radio-group";
+import * as Tooltip from "@radix-ui/react-tooltip";
+import type { FC } from "react";
+import { AiOutlineMore } from "react-icons/ai";
+import { HiCheckCircle } from "react-icons/hi";
+// import resolveConfig from "tailwindcss/resolveConfig";
+
+import { trpc } from "../../features/trpc/hooks";
+// import tailwindConfig from "../../../tailwind.config";
+
+// const fullConfig = resolveConfig({
+//   ...tailwindConfig,
+//   content: ["./src/**/*.{html,js,ts,tsx}"],
+// });
+
+// export const themeColors = fullConfig.theme?.colors as Record<
+//   EventColorUserRadix & string,
+//   Record<number, string>
+// >;
```

Comment:
the addin doesn't like importing here and panics if i uncomment this.

guessing this is also why i cant mount the calendar since it does a similar thing

![CleanShot 2022-12-03 at 14 58 42](https://user-images.githubusercontent.com/51714798/205444497-e520f054-3e89-42c8-beeb-7d23c3351ae6.png)